packages/plugins/src/hardhat/deployments.ts

```diff
@@ -47,7 +47,7 @@ export const deployAllChugSplashConfigs = async (
   silent: boolean,
   ipfsUrl: string,
   fileNames?: string[]
-) => {
+): Promise<boolean> => {
```

Comment:
What's the purpose of returning a boolean here? It doesn't look like we ever return false